hadoop-hdds/rocks-native/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedSSTDumpIterator.java

```diff
@@ -58,9 +60,16 @@ public abstract class ManagedSSTDumpIterator<T> implements ClosableIterator<T> {
   private AtomicBoolean open;
   private StackTraceElement[] stackTrace;
 
+  public ManagedSSTDumpIterator(ManagedSSTDumpTool sstDumpTool,
```

Comment:
This alignment is not correct as per Ozone coding guideline here and as well as many other places changed in this PR.

It would be great if you can configure your IDE to [Ozone-style](https://github.com/apache/ozone/blob/master/hadoop-ozone/dev-support/intellij/ozone-style.xml)

Reply:
This is not addressed and I don't know why your IDE is not auto indenting this if you have configured [Ozone-style](https://github.com/apache/ozone/blob/master/hadoop-ozone/dev-support/intellij/ozone-style.xml).

To me, it looks like it is configured and it does indent because line 70-73 are properly indented.

Reply:
I am not sure why reimported it. It was able to indent it properly. I guess something changed in my settings. It should be fine going forward.